snap/db.go

```diff
@@ -60,15 +63,19 @@ func (s *Snapshotter) SaveDBFrom(r io.Reader, id uint64) (int64, error) {
 // DBFilePath returns the file path for the snapshot of the database with
 // given id. If the snapshot does not exist, it returns error.
 func (s *Snapshotter) DBFilePath(id uint64) (string, error) {
-	fns, err := fileutil.ReadDir(s.dir)
+	return GetDBFilePathByID(s.dir, id)
+}
+
+func GetDBFilePathByID(dbPath string, id uint64) (string, error) {
```

Comment:
I needs to export this function so that etcdserver can use it to find xxx.snap.db.